app-web/src/components/Home/ResourcePreview.js

```diff
@@ -127,11 +124,50 @@ export const ResourcePreview = ({ title, link, resources, filters, amountToShow,
     }
     //reset the amount of resources to show
     setCount(amountToShow);
+    setActiveFilters(newPillList);
   };
 
   let resultLabel = getSearchResultLabel(resources.length);
   resultLabel = resources.length !== 0 ? 'All ' + resultLabel : resultLabel;
   let pills = [];
+  // Method 2 with used state
+  const addtourl = () => {
+    // Valid filter name and it's not selected yet!
+    const url = new URL(window.location.href);
+    const searchQuery = url.searchParams.get('q');
+    const stringified = queryString.stringify(
```

Comment:
I think this could be simplified. `queryString` has a parse method. 

`const searchParams = queryString.parse(location.search)`
`const newSearchParams = {...searchParams, filters: activeFIlters}`